docs/sdk-and-tools/indices/blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+id: blocks
+title: blocks
+---
+
+The `_id` field for this index is composed of block hash hex encoded
```

Comment:
The `_id` field of this index is represented by the block hash, in a hexadecimal encoding

Reply:
done